config/hardening/ssg-supplemental.sh

```diff
@@ -294,52 +292,58 @@ cat <<EOF > /etc/audit/rules.d/audit.rules
 -w /var/log/btmp -p wa -k session
 -w /var/log/wtmp -p wa -k session
 
+#2.6.2.4.12 Ensure auditd Collects System Administrator Actions
+-w /etc/sudoers -p wa -k actions
+
+#2.6.2.4.13 Make the auditd Configuration Immutable
+-w /sbin/insmod -p x -k modules
+-w /sbin/rmmod -p x -k modules
+-w /sbin/modprobe -p x -k modules
+-a always,exit -F arch=b32 -S init_module -S delete_module -k modules
+-a always,exit -F arch=b64 -S init_module -S delete_module -k modules
+
+# Ignore all the anonymous events. Often tagged on every rule, but ignoring 
```

Comment:
In the PR, it shows the old line was removed:

`````
-a always,exit -F path=/usr/sbin/semanage -F perm=x -F auid>=1000 -F auid!=4294967295 -F key=privileged-priv_change
`````

And replaced with:
`````
+-a always,exit -F path=/usr/sbin/semanage -F perm=x -F key=privileged-priv_change
`````

(without the auid tags)